app/views/shelter_reviews/new.html.erb

```diff
@@ -0,0 +1,21 @@
+<h2>
+  <%= form_tag "/shelters/#{@shelter.id}/reviews" do %>
```

Comment:
We could also use a form_with syntax

Reply:
That is true but the rest of the forms follow this format so I think it makes the most sense to be consistent. We can decide on our final approach during a refactor.